src/Layout/Box/BoxStoryStyle.tsx

```diff
@@ -0,0 +1,16 @@
+import styled from 'styled-components';
+
+import { Box as BaseBox } from '../../../src/Layout/Box';
+
+export const GreyBackground = styled.div`
+  display: inline-block;
```

Comment:
@rosslo , as the styles are all like one property added, how about we add the `style` prop inside the div tag and include it in the Stories JSX.
Else in the documentation's source code, it will be something like <S.Content> etc which might make less sense?

Reply:
Oh I thought specific JSX component name will be easier to understand without reading the inline-style. Wdyt?
![image](https://user-images.githubusercontent.com/13249748/105961560-9fbb4a80-60b9-11eb-9454-c99a72ca6e98.png)